cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsPlatformResources.java

```diff
@@ -841,50 +842,49 @@ public CloudEncryptionKeys encryptionKeys(ExtendedCloudCredential cloudCredentia
+                                    .ifPresent(item -> {
+                                        DescribeKeyRequest describeKeyRequest = new DescribeKeyRequest().withKeyId(item.getKeyId());
+                                        DescribeKeyResult describeKeyResult = client.describeKey(describeKeyRequest);
+                                        Map<String, Object> meta = new HashMap<>();
+                                        meta.put("aWSAccountId", describeKeyResult.getKeyMetadata().getAWSAccountId());
+                                        meta.put("creationDate", describeKeyResult.getKeyMetadata().getCreationDate());
+                                        meta.put("enabled", describeKeyResult.getKeyMetadata().getEnabled());
+                                        meta.put("expirationModel", describeKeyResult.getKeyMetadata().getExpirationModel());
+                                        meta.put("keyManager", describeKeyResult.getKeyMetadata().getKeyManager());
+                                        meta.put("keyState", describeKeyResult.getKeyMetadata().getKeyState());
+                                        meta.put("keyUsage", describeKeyResult.getKeyMetadata().getKeyUsage());
+                                        meta.put("origin", describeKeyResult.getKeyMetadata().getOrigin());
+                                        meta.put("validTo", describeKeyResult.getKeyMetadata().getValidTo());
+
+                                        if (!CloudConstants.AWS.equalsIgnoreCase(describeKeyResult.getKeyMetadata().getKeyManager())) {
+                                            CloudEncryptionKey key = new CloudEncryptionKey(
+                                                    item.getKeyArn(),
+                                                    describeKeyResult.getKeyMetadata().getKeyId(),
+                                                    describeKeyResult.getKeyMetadata().getDescription(),
+                                                    keyListEntry.getAliasName().replace("alias/", ""),
+                                                    meta);
+                                            cloudEncryptionKeys.getCloudEncryptionKeys().add(key);
+                                        }
+                                    });
+                        } catch (AmazonServiceException e) {
+                            if (e.getStatusCode() == UNAUTHORIZED) {
+                                String policyMessage = "Could not get encryption keys because the user does not have enough permission.";
+                                LOGGER.error(policyMessage, e);
+                            } else {
+                                LOGGER.info(queryFailedMessage, e);
+                            }
+                        } catch (Exception e) {
+                            LOGGER.warn(queryFailedMessage, e);
+                        }
```

Comment:
Nit: This try-catch block isn't indented properly. Also, could this be added as a standalone method considering there is quite a bit of logic being executed.